README.rst

```diff
@@ -29,7 +29,7 @@ http://temboard.io/ for the big picture.
 .. code-block::
 
    pip install temboard-agent
-
+   sudo -u postgres temboard-agent -c /usr/local/share/temboard-agent/quickstart/temboard-agent.conf
```

Comment:
It is mapped to `data_files` from `setup.py`:

``` python
setup(
    # ...
    data_files=[
        ('share/temboard-agent/quickstart/', [
            'share/temboard-agent.conf',
            'share/temboard-agent_CHANGEME.pem',
            'share/temboard-agent_CHANGEME.key',
            'share/temboard-agent_ca_certs_CHANGEME.pem',
            'share/temboard-agent.logrotate'
        ]),
        ('lib/systemd/system', ['temboard-agent.service']),
    ])
```

Do you have a different installation ?